Add docs for {today} token (#343)
mautic/mautic-documentation: pages/09.Channels/02.Emails/01.Managing emails/docs.en.md

- Default format for date only  
- Default Time Only Format 

### Today token

> Token `{today}` display actual date time to the email. By default, you can use predefined formats after `|` symbol:

Reviewer: ```suggestion
The token `{today}` displays the actual date and time in the email or landing page. By default, you can use predefined formats after the `|` symbol:
```

> - `{today}` same as `{today|datetime}`

Reviewer: ```suggestion
- `{today}` uses the same format as `{today|datetime}`
```

- `{today|date}`
- `{today|time}`

> Your date will display as human reading format taken from  Configuration > System Settings

Reviewer: ```suggestion
The date displays in a human readable format, taken from  Configuration > System Settings
```

- Default format for date only
- Default Time Only Format

#### Custom format 

> Returns the formatted date string based on PHP https://www.php.net/manual/en/datetime.format.php

Reviewer: ```suggestion
Returns the formatted date string based on PHP datetime format supplied - see [documentation][php-datetime]
```

Please add a macro [php-datetime] at the bottom of the page linking to https://www.php.net/manual/en/datetime.format.php

- `{today|Y-m-d H:i:s}`
- `{today|d. m. Y}` 

#### Relative date

> You can use relative date after second `|` symbol:

Reviewer: ```suggestion
You can use the Today token to show a relative date by using a second `|` symbol:
```
